src/main/java/triss/Triss.java

```diff
@@ -0,0 +1,429 @@
+import triss.task.Deadline;
+import triss.task.Event;
+import triss.task.Task;
+import triss.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Triss {
+
+    /** Logo shown during startup */
+    private static final String LOGO = "████████ ██████  ██ ███████ ███████ \n" +
+            "   ██    ██   ██ ██ ██      ██      \n" +
+            "   ██    ██████  ██ ███████ ███████ \n" +
+            "   ██    ██   ██ ██      ██      ██ \n" +
+            "   ██    ██   ██ ██ ███████ ███████ \n";
+
+    /** String of underscores to separate user input and Triss output */
+    public static final String SEPARATOR_LINE = "____________________________________________________________";
+
+    /** Boolean to track if user has said "bye" */
+    private static boolean hasUserSaidBye = false;
+
+    /** Array to keep track of user's tasks */
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+
+    /** Length of the word "todo" */
+    public static final int END_INDEX_OF_WORD_TODO = 4;
+    /** Length of the word "deadline" */
+    public static final int END_INDEX_OF_WORD_DEADLINE = 8;
+    /** Length of the word "event" */
+    public static final int END_INDEX_OF_WORD_EVENT = 5;
+
+    /**
+     * Loops Triss into receiving user input and giving output messages.
+     * It will only end once the user types "bye".
+     * @param args No params currently used.
+     */
+    public static void main(String[] args) {
+        // Print LOGO and welcome text
+        printWelcomeMessage();
+
+        // Initialise user input reader
+        Scanner in = createNewInputReader();
+
+        // Check for any stored data
+        try {
+            initialiseDataStorage();
+        } catch (TrissException error) {
+            System.out.println(error.getMessage());
+        } catch (IOException error) {
+            System.out.println("Tasks storage has been corrupted! Try restarting.");
+        }
+
+        // While user has not said "bye", check for next line of input
+        while (!hasUserSaidBye) {
+
+            // Get the next line of input, and parse it to find the user's command (first word in input)
+            String userInput = getUserInput(in);
+            String userCommand = parseUserInput(userInput, 0);
+            printLine(SEPARATOR_LINE);
+
+            // Perform actions based on user's command
+            switch (userCommand) {
+            case "bye":
+                handleUserShuttingDown();
+                printShutdownMessage();
+                break;
+            case "list":
+                printAllTasks();
+                break;
+            case "done":
+                handleUserMarkingTaskAsDone(userInput);
+                break;
+            case "delete":
+                handleUserDeletingTask(userInput);
+                break;
+            default:
+                try {
```

Comment:
Perhaps a better/ more clear implementation would be to have a case for the respective creatingTask commands so that the default case can throw an error straight.

Reply:
Roger that, thanks.